build.rs

```diff
@@ -12,4 +12,16 @@ fn main() {
     {
         println!("cargo:rustc-link-lib=static=pam");
     }
+
+    generate_build_info();
+}
+
+// uses the 'built' crate to generate a build.rs file with a bunch of build information. We then
+// include this file in the app module.
+fn generate_build_info() {
+    let src = env::var("CARGO_MANIFEST_DIR").unwrap();
+    let dst = std::path::Path::new(&env::var("OUT_DIR").unwrap()).join("build-info.rs");
```

Comment:
Is it necessary to do `&env::var`? In other `build.rs` scripts I just use `env::var`

Reply:
Ow as you do a line above :P